testfixtures/tests/test_popen.py

```diff
@@ -57,7 +58,7 @@ def test_command_max_args(self):
 
     def test_callable(self):
         def some_callable(command, stdin):
-            return bytes(command, encoding='utf8'), bytes(stdin, encoding='utf8'), 1, 345, 0
+            return six.b(command), six.b(stdin), 1, 345, 0
```

Comment:
I think you'd be looking for https://github.com/Simplistix/testfixtures/blob/master/testfixtures/compat.py#L13.
I've managed to avoid depending on six so far...